src/parser/vp9_parser.cpp

```diff
@@ -269,18 +390,51 @@ ParserResult Vp9VideoParser::FindFreeInDecBufPool() {
     }
     curr_pic_.dec_buf_idx = dec_buf_index;
     decode_buffer_pool_[dec_buf_index].use_status |= kFrameUsedForDecode;
-    // Todo decode_buffer_pool_[dec_buf_index].pic_order_cnt = curr_pic_.order_hint;
     decode_buffer_pool_[dec_buf_index].pts = curr_pts_;
     return PARSER_OK;
 }
 
 ParserResult Vp9VideoParser::FindFreeInDpbAndMark() {
     int i;
-    /*Todo*/
-
+    for (i = 0; i < VP9_NUM_REF_FRAMES; i++ ) {
+        if (dpb_buffer_.dec_ref_count[i] == 0) {
```

Comment:
suggest combining this with the condition for for loop

Reply:
Prefer the current code since it looks cleaner.